### PR TITLE
nav.jsx: change the name and highlight component when a user clicks

### PR DESCRIPTION
**Pull request by checkmate-bitch**
The heading and highlight component changes dynamically according to user clicks

### CODE REVIEW

File: app/components/nav.jsx [hunk 1]
```diff
@@ -2,23 +2,19 @@ var React = require('react')
...
-
   getLi: function () {
+    // console.log("props",this.props.highlight);
+
+    // make individual li component for ul
     return this.props.components.map( (comp,i) => {
-      return <li id={comp} key={i} className={ this.state.isActiveLiNo === i ? "active" : "" } 
-              onClick={ () => this.clickHandler(i) }>
-                <a href="#">{comp}</a>
+      return <li id={comp} data-key={i} className={ this.props.highlight === i ? "active" : "" }
```
Comment (tamj0rd2) [inline app/components/nav.jsx]: Why did you change this to data-key?

Comment (tamj0rd2) [inline app/components/nav.jsx:15]: Why did you change this to data-key? `key` on it's own is what React uses :)

Comment (checkmate-bitch) [reply app/components/nav.jsx:15]: IDK. I'm not used to react yet. JS uses data-key. Thought it might be useful

Comment (tamj0rd2) [reply app/components/nav.jsx:15]: Ah right, you may as well take it out if you're not using it for anything in particular.

If react gives an error about a Key being needed, change it back to `key=`
